mne_bids/tests/test_read.py

```diff
@@ -133,6 +139,90 @@ def test_handle_events_reading():
     events, event_id = mne.events_from_annotations(raw)
 
 
+def test_line_freq_estimation():
+    """Test estimating line frequency when
+    reading a BIDS sidecar.json file."""
```

Comment:
should be one line docstring. You don't have pep8 checker on your editor?

Reply:
Hmmm... that's weird, it passed:

```
(mne) adam2392@Adams-MacBook-Pro-3 mne-bids % make pep
Running flake8
0
flake8 passed
Running pydocstyle
check-manifest --ignore .circleci*,doc,.DS_Store
lists of files in version control and sdist match
```

I'll fix this tho.

Reply:
I don't think the test folder is included in the `make pep`. Can you check by deliberately making a pep8 fault and seeing if it's caught? 

I'm not exactly sure why we don't enforce it there, I think it was copied from what mne-python does. But I think it's good practice to adhere to it anyway when possible.

Reply:
I tried and yes it doesn't raise the error because it is not checking the tests/ directory at all. I added it into the `Makefile`.